utils/strings.go

```diff
@@ -79,3 +80,14 @@ func LongestCommonPrefix(list []string) string {
 	}
 	return list[0]
 }
+
+func StringHasher() func(str string) uint64 {
+	seed := maphash.MakeSeed()
```

Comment:
We may need a way to re-instantiate a new seed.

Reply:
I loved your solution, very simple! But without fixing this re-seed, we cannot use it, or else when we refresh the Random Albums page we get the same result.... We have to figure this out.

Reply:
Yeah I agree. Just not sure what approach we want to take here to get us closer to the ideal behavior.
A pushed a new commit to reseed at the "first page" of the request for random albums.
It should be enough to fix the issue you mentioned and the one linked in the PR description.